internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -395,6 +416,10 @@ type networkLoadBalancerConfig struct {
 	appDNSName           *string
 }
 
+type applicationLoadBalancerConfig struct {
+	settings *template.ALBListener
+}
```

Comment:
I agree with @iamhopaul123's earlier feedback that we can follow YAGNI for now and get rid of this struct in favor of `*template.ALBListener`. We can introduce it in the future when we need it.